src/views/index.html

```diff
@@ -77,6 +77,7 @@
     <nav id="main-menu" class="hidden">
       <ul>
         <li><a id="things-menu-item" href="/things" class="selected">Things</a></li>
+        <li><a id="rules-engine-menu-item" href="/rules-engine">Rules Engine</a></li>
```

Comment:
Thanks for the icon! What are your thoughts on how to best approach hiding this when the rules engine isn't enabled?